pkg/tso/keyspace_group_manager_test.go

```diff
@@ -109,15 +109,15 @@ func (suite *keyspaceGroupManagerTestSuite) TestDeletedGroupCleanup() {
 	suite.applyEtcdEvents(re, []*etcdEvent{generateKeyspaceGroupPutEvent(1, []uint32{1}, []string{svcAddr})})
 	// Check if the TSO key is created.
 	testutil.Eventually(re, func() bool {
-		ts, err := mgr.storage.LoadTimestamp(keypath.Prefix(keypath.TimestampPath(1)))
+		ts, err := mgr.storage.LoadTimestamp(keypath.TimestampPath(1))
```

Comment:
Do we need to add some other test to avoid the similiar problems?

Reply:
I think https://github.com/tikv/pd/pull/9126 will be a stronger test to avoid this.

Reply:
BTW, the reason that we do not detect the problem of `LoadTimestamp` is the non-timestamp key has been continue in this. So it will not cause any actual damage. 😂

https://github.com/tikv/pd/blob/e245c1fb30c9ccac07d30f813b2bbd486f31c058/pkg/storage/endpoint/tso.go#L56-L60